rdmd_test.d

```diff
@@ -140,6 +140,18 @@ void runCompilerAgnosticTests(string rdmdApp, string defaultCompiler, string mod
     assert(res.status == 0, res.output);
     assert(res.output.canFind("Usage: rdmd [RDMD AND DMD OPTIONS]... program [PROGRAM OPTIONS]..."));
 
+    string helpText = res.output;
+
+    // verify help text matches expected defaultCompiler
+    {
+        enum compilerHelpLine = "  --compiler=comp    use the specified compiler (e.g. gdmd) instead of ";
+        auto offset = helpText.indexOf(compilerHelpLine);
+        assert(offset >= 0);
+        auto compilerInHelp = helpText[offset + compilerHelpLine.length .. $];
+        compilerInHelp = compilerInHelp[0 .. compilerInHelp.indexOf('\n')];
+        assert(defaultCompiler.baseName == compilerInHelp);
+    }
```

Comment:
This is a nice additional test, thanks for adding it.

However, if you're thinking that it provides a path to (in future) removing the need for the `--rdmd-default-compiler` flag ... it won't.  See remarks in the earlier PR about why that is.